### ISSUE TITLE
bitbucket #3: initialize password for Django superuser

### ISSUE THREAD

**Issue by dongbohu** [greenelab/adage-server#226]
Copied from bitbucket: 
https://bitbucket.org/greenelab/adage-server/issues/3/initialize-password-for-django-superuser

The `create_admin_user` task in `fabfile/adage_server.py` does not set an initial password because it is not clear how to do so non-interactively. This should be part of this step or the superuser should be disabled until a password is created."

**Comment by cgreene** [comment]
Should we just disable the superuser? We don't use any of django's user or admin functionality.

**Comment by dongbohu** [comment]
I think `superuser` can be disabled. What do you think, @mhuyck?

**Comment by mhuyck** [comment]
Agreed. When I put this in I was thinking it would be used for data loading or something but we ended up just using management commands. If we go down the path of maintaining user accounts then we may need a superuser for that but we could always re-enable it then.
